apps/dashboard/src/contract-ui/tabs/tokens/components/airdrop-upload-erc20.tsx

```diff
@@ -402,13 +402,13 @@ const AirdropTable: React.FC<AirdropTableProps> = ({ data, portalRef }) => {
               <IconButton
```

Comment:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/yNOf1svJ8o3zjO7zQouZ/7ddb11c9-3612-4868-9be9-779b574d0ffa.png)

This is not related to changed done in this PR - but let's fix the text alignment of "Page x of x" here as well